packages/legacy/core/App/types/ScreenOptions.ts

```diff
@@ -0,0 +1,37 @@
+import { StackNavigationOptions } from "@react-navigation/stack";
+
+export interface ScreenOptions {
```

Comment:
I think this will be a redundant re-declaration. There is already ScreenOptionsType exported, you just need to refactor to make it apply to all screens instead of OnboardingStack and rename the Token. 
That's not to mention that the naming would conflict with Global Layout options which is in this PR #1321